src/lib/cache/redis/redis.go

```diff
@@ -107,23 +91,45 @@ func New(opts cache.Options) (cache.Cache, error) {
 		opts.Address = "redis://localhost:6379/0"
```

Comment:
since we hardcode 0 here, do we mean that only core can use it without specify address?

Reply:
yes, but in normal the address will not be empty.